cmd/mattermost/commands/channel.go

```diff
@@ -99,11 +99,11 @@ Channels can be specified by [team]:[channel]. ie. myteam:mychannel or by channe
 }
 
 var ModifyChannelCmd = &cobra.Command{
-	Use:   "modify [channel]",
+	Use:   "modify [channel] [flags]",
 	Short: "Modify a channel's public/private type",
 	Long: `Change the public/private type of a channel.
 Channel can be specified by [team]:[channel]. ie. myteam:mychannel or by channel ID.`,
-	Example: "  channel modify myteam:mychannel --private",
+	Example: "  channel modify myteam:mychannel --private --username user-1",
```

Comment:
Propose `myusername` instead of `user-1`. More consistent with `myteam` and `mychannel`

Reply:
done